src/toast/ops/pointing_healpix.py

```diff
@@ -253,26 +241,21 @@ def _exec(self, data, detectors=None, **kwargs):
                 continue
 
             # Do we already have pointing for all requested detectors?
-            done = False
             if (self.pixels in ob.detdata) and (self.weights in ob.detdata):
-                done = True
                 pix_dets = ob.detdata[self.pixels].detectors
                 wt_dets = ob.detdata[self.weights].detectors
                 for d in dets:
                     if d not in pix_dets:
-                        done = False
                         break
                     if d not in wt_dets:
-                        done = False
                         break
-            if done:
-                # We already have pointing for all specified detectors
-                if data.comm.group_rank == 0:
-                    msg = "Group {}, ob {}, pointing already computed for {}".format(
-                        data.comm.group, ob.name, dets
-                    )
-                    log.verbose(msg)
-                continue
+                else:
```

Comment:
Fascinating. I trust this is like the list comprehensions that quickly grow on you. I added the `# no break` comments to the `else` clauses as the SO post suggested. In this case, the loop can never be empty because we check it explicitly above but even it if was, `continue` would be the right action in such case.